src/sentry/eventstream/snuba.py

```diff
@@ -42,3 +42,9 @@ def start_unmerge(self, project_id, hashes, previous_group_id, new_group_id):
 
     def end_unmerge(self, state):
         pass
+
+    def start_delete_tag(self, project_id, tag):
+        pass
+
+    def end_delete_tag(self, state):
+        pass
```

Comment:
Is there a plan to expose this via the HTTP API (like `insert_raw` above) for local development?

Reply:
Yeah, we should probably do all of them (delete_group, merge, unmerge, delete_tag). I may take a looksee tomorrow.

Reply:
https://github.com/getsentry/snuba/pull/274